packages/notifier/test/iterable-testing-tools.js

```diff
@@ -4,6 +4,7 @@ import { E } from '@agoric/eventual-send';
 import { makePromiseKit } from '@agoric/promise-kit';
 import { observeIteration, observeIterator } from '../src/index';
 
+import '@agoric/marshal';
```

Comment:
This should be:

```suggestion
import '@agoric/marshal/exported';
```

so that the marshal library's code is not dragged in.

Reply:
Done.